pkg/plugins/pluggable/loader.go

```diff
@@ -133,6 +140,58 @@ func (l *PluginLoader) Load(pluginType PluginTypeConfig) (interface{}, func(), e
 	return raw, cleanup, nil
 }
 
+func (l *PluginLoader) setUpDebugger(client *plugin.Client, cleanup func()) (func(), error) {
+	debugContext := l.Context.PlugInDebugContext
+	if len(debugContext.RunPlugInInDebugger) > 0 && strings.ToLower(l.SelectedPluginKey.String()) == strings.TrimSpace(strings.ToLower(debugContext.RunPlugInInDebugger)) {
```

Comment:
This line seems to be susceptible to a panic (seen from the [integration test](https://dev.azure.com/deislabs/porter/_build/results?buildId=6909&view=logs&j=0d1d2f05-4482-5088-4d55-986bcf02e524&t=bd076812-b150-58c7-9f90-9c2fa032ad9b)):

```
panic(0x2298ea0, 0x3738610)
	/opt/hostedtoolcache/go/1.13.10/x64/src/runtime/panic.go:679 +0x1b2
get.porter.sh/porter/pkg/plugins/pluggable.(*PluginLoader).setUpDebugger(0xc00036a8e0, 0xc00049a000, 0xc00046e610, 0x0, 0x0, 0x0)
	/home/vsts/work/1/s/pkg/plugins/pluggable/loader.go:145 +0x4b
```

Reply:
Arghhh will take a look